lib/Mojolicious/Static.pm

```diff
@@ -33,6 +35,9 @@ sub dispatch {
   $path = $stash->{path} ? $path->new($stash->{path}) : $path->clone;
   return undef unless my @parts = @{$path->canonicalize->parts};
 
+  # Don't serve when a part has embedded path separators on windows
+  if (IS_WINDOWS) { m!\\! && return for @parts };
```

Comment:
The bare return is wrong, since it's context sensitive. We consider that a security issue.

Reply:
Also, why not `/\\/`?